tomoj/protocols/protocol_xCorrPrealignment.py

```diff
@@ -54,39 +58,132 @@ def _defineParams(self, form):
-                      display=params.EnumParam.DISPLAY_HLIST,
-                      help='Generate and save the interpolated tilt-series applying the'
-                           'obtained transformation matrices.')
-
-        group = form.addGroup('Interpolated tilt-series',
-                              condition='computeAlignment==0')
-
-        group.addParam('binning', params.FloatParam,
-                       default=1.0,
-                       label='Binning',
-                       help='Binning to be applied to the interpolated tilt-series. '
-                            'Must be a integer bigger than 1')
-
-        form.addParam('rotationAngle',
-                      params.FloatParam,
-                      label='Tilt rotation angle (deg)',
-                      default='0.0',
-                      expertLevel=params.LEVEL_ADVANCED,
-                      help="Angle from the vertical to the tilt axis in raw images.")
+                      label='Multiscale cross-correlation',
+                      important=True,
+                      help='Apply a multiscale approach with the given number '
+                           'of level. (Default = 1 = no multiscale')
+
+        form.addParam('cumulativereference', params.BooleanParam,
+                      default=False,
+                      label='Cumulative reference',
+                      important=True,
+                      help='If true, the processing is not done between '
+                           'consecutive images but using central image as '
+                           'reference to which is added the newly aligned '
+                           'images.')
+
+        form.addParam('loop', params.BooleanParam,
+                      default=False,
+                      label='Loop until stabilization',
+                      important=True,
+                      help='Refine alignment by doing the cross-correlation as'
+                           ' many times as needed to stabilize.')
+
+        form.addParam('computeAlignment', params.BooleanParam,
+                      default=False,
+                      label='Generate interpolated tilt-series',
+                      important=True,
+                      help='Generate and save the interpolated tilt-series '
+                           'applying the obtained transformation matrices.')
+
 
     # -------------------------- INSERT steps functions ---------------------
     def _insertAllSteps(self):
         for ts in self.inputSetOfTiltSeries.get():
             self._insertFunctionStep('convertInputStep', ts.getObjId())
             self._insertFunctionStep('computeXcorrStep', ts.getObjId())
-            if self.computeAlignment.get() == 0:
-                self._insertFunctionStep('computeInterpolatedStackStep', ts.getObjId())
+            # if self.computeAlignment.get() == 0:
+            #     self._insertFunctionStep('computeInterpolatedStackStep', ts.getObjId())
```

Comment:
The generation of the interpolated tilt series should be available in the final version. If tomoJ does not have a method to apply transformation matrices, this operation can be done with ImageHandler class from Scipion